src/elmFormat.ts

```diff
@@ -76,11 +78,14 @@ function elmFormat(document: vscode.TextDocument) {
     'elm',
   );
   const formatCommand: string = <string>config.get('formatCommand');
+  const dummyPath = path.join(vscode.workspace.rootPath, 'dummyfile');
+  const [_, elmVersion] = utils.detectProjectRootAndElmVersion(dummyPath, vscode.workspace.rootPath);
+  const args = utils.isElm019(elmVersion) ? ['--stdin', '--yes'] : ['--stdin', '--elm-version 0.18', '--yes'];
```

Comment:
FYI, starting in elm-format 0.8.1, There the elm-format binary will no longer default to Elm 0.19--instead it will check the cwd for an `elm.json` file or `elm-package.json` file to determine whether to default to Elm 0.18 or 0.19 mode.  If it finds neither in the cwd, then it will fail with an error.   So I'm thinking it might be safer and more robust here to always add the `--elm-version=0.19` flag when the plugin thinks it's elm 0.19?

Reply:
Thanks for the heads up - fixed in #272